common/src/config.rs

```diff
@@ -212,6 +215,7 @@ struct Limits {
 impl Default for Limits {
     fn default() -> Self {
         Limits {
+            max_active_events: 20,
```

Comment:
This should certainly be in the 100s. An event can easily take 300ms to be processed and sent to sentry. With 20 we would throttle to 69 events per second with is almost a bit too low. 

Apart from that, we need to be very careful when choosing this parameter in production since it causes events to get unconditionally dropped.